public/index.html

```diff
@@ -185,6 +185,52 @@
 
     <hr/>
 
+		<uxtabarea>
+			<uxtablinkcontainer>
+				<uxtablink controls="firstTab" active="true">Tab One</uxtablink>
+				<uxtablink controls="secondTab">Tab Two</uxtablink>
+				<uxtablink controls="thirdTab">Tab Three</uxtablink>
```

Comment:
Agree - we can say it's a convention that the tab link order must match the tab pane order. Unless we have a use case for it. Kinda like the TDD approach, we don't add the feature until we need it?